kitsune/wiki/urls.py

```diff
@@ -161,3 +161,14 @@
         r"^/discuss/watch_locale$", kbforums_views.watch_locale, name="wiki.discuss.watch_locale"
     ),
 ]
+
+urlpatterns += [
```

Comment:
So, I believe this should go under products and we should account for more cases.

* A user visits `https://help.getpocket.com/`. We should get a redirect with no params landing users to the pocket product page.
* A user visits `https://help.getpocket.com/category/857-category`. We should get the `category` and `857-category` as params. During processing, we are going to ignore those params and redirect to the product landing page.
* A user visits anything in the `help.getpocket.com/` domain that is not followed by `article`, we should redirect to the product landing page.
* If and only if there is url of the form `help.getpocket.com/article/number-slug` then we are extracting the slug part and we are redirecting to the KB page if exists. If not we are redirect to the product landing page.

Reply:
By going under products, I mean that we should aim at `https://support.mozilla.org/en-US/products/pocket`. That url needs to accept optional parameters that we need to unpack and do the processing.

Alternatively we can ask our SREs to redirect everything to `https://support.mozilla.org/en-US/products/pocket` and `help.getpocket.com/article/*` to a view similar with the one you created. I would choose the path of least resistance. If SREs can help us with two redirects, we can optimize what we have here

Reply:
I assume that refractr will take a root url request and/or anything not an article and send it directly to '/products/pocket', and then my code handles articles and fails back gracefully to product. I think this is the path of least resistance, right? So your use cases - `/category/857-category` is going to just redirect right from refractr to `/products/pocket`, as will 'help.getpocket.com`.  If that's the case, I don't think we should move the view under products because it really is just a `kb` view.

Reply:
@akatsoulas Did I sway youabout leaving it under wiki, or do you still prefer I move the view to products ?